lesson19/src/js/index.js

```diff
@@ -0,0 +1,74 @@
+import { createAttributedElements } from './utiles/createAttributeWithEllement';
+import { showLoadingImage, removeLoading } from './modules/loading';
+import { addSliderContents } from './slider';
+import { addTabContents, changeTabs} from './news';
+
```

Comment:
I thought the import and export were nicely utilized and good. I learned a lot!

Reply:
Thank you!
I was also talking about modularization by Mr. Monaka and my teacher, so I tried to incorporate it.